site-src/v1alpha2/guides/multiple-ns.md

```diff
@@ -1,158 +1,163 @@
-their requirements:
-
-- The site team has two applications, _home_ and _login_, that are running 
-behind `foo.example.com`. They want to isolate access and configuration across
-their apps as much as possible to minimize access and failure domains. 
-They use separate HTTPRoutes for each app, to isolate app routing configurations 
-such as canary rollouts. but share the same load balancer 
-IP, port, domain, and TLS certificate.
-- The store team has a single Service called _store_ that they have deployed 
-in the `store-ns` Namespace. 
-- The Foobar Corporation operates behind the `foo.example.com` domain so they 
-would like to host all applications on the same Gateway resource. This is 
-controlled by a central infrastructure team, operating in the `infra-ns` Namespace.
-- Lastly, the security team controls the certificate for `foo.example.com`. 
-By managing this certificate through the single shared Gateway they are able 
+in the same Kubernetes cluster in the `store-ns` and `site-ns` Namespaces. These
+are their goals and how they use Gateway API resources to accomplish them:
+
+- The site team has two applications, _home_ and _login_. The team wants to to isolate access and 
+configuration across their apps as much as possible to minimize access and failure domains.
+They use separate HTTPRoutes attached to the same Gateway to isolate routing configurations,
+such as canary rollouts, and still share the same but share the same IP address,
+ port, DNS domain, and TLS certificate.
+- The store team has a single Service called _store_ that they have deployed
+in the `store-ns` Namespace which also needs to be exposed behind the same IP
+ address and domain.
+- The Foobar Corporation operates behind the `foo.example.com` domain for all
+ apps. This is controlled by a central infrastructure team, operating in the
+  `infra-ns` Namespace.
+- Lastly, the security team controls the certificate for `foo.example.com`.
+By managing this certificate through the single shared Gateway they are able
 to centrally control security without directly involving application teams.
 
 The logical relationship between the Gateway API resources looks like this:
 
 ![Cross-Namespace routing](../images/cross-namespace-routing.svg)
 
-## Cross-namespace Route binding
+## Cross-namespace Route Attachment
 
-[Route binding](/concepts/api-overview/#route-binding) is an important concept 
-that dictates how Routes and Gateways select each other to apply routing
-configuration to a Gateway. It is especially relevant when there are multiple
-Gateways and multiple Namespaces in a cluster. Gateway and Route binding is
-bidirectional - a binding can only exist if the Gateway owner and Route owner
-owner both agree to the relationship. This bi-directional relationship exists
-for two reasons:
+[Route attachment][attaching] is an important concept that dictates how Routes
+and Gateways select each other to apply routing configuration to a Gateway. It
```

Comment:
I think it's better to remove the idea of selecting each other since that's not really what happens now.
```suggestion
are attached to Gateways. It
```